### PR TITLE
refactor(auth): add back to login button for some login steps FR-1170

### PR DESCRIPTION
**Pull request by AntonReact**

### CODE REVIEW

**Comment by AntonReact** [comment]
https://frontegg.atlassian.net/browse/FR-1170

**Review by MaxArnaut — APPROVED** [review]

File: packages/auth/src/Login/ForceEnrollMfa.tsx [hunk 1]
```diff
@@ -112,7 +110,7 @@ export const ForceEnrollMfa: FC<ForceEnrollMfaProps> = (props) => {
               <FButton
                 type='submit'
                 size='large'
-                fullWidth={false}
+                fullWidth={true}
```
Comment (frontegg-david) [inline packages/auth/src/Login/ForceEnrollMfa.tsx:113]: why making the button full width?

**Review by frontegg-david — APPROVED** [review]

File: packages/auth/src/Login/ForceEnrollMfa.tsx (hunk 1 above)
Comment (frontegg-david) [reply packages/auth/src/Login/ForceEnrollMfa.tsx:113]: @AntonReact

Comment (AntonReact) [reply packages/auth/src/Login/ForceEnrollMfa.tsx:113]: Screen Shot 2021-01-15 at 9 49 37 AM
I've added "back to login" button everywhere. Have changed full width to make everything looks the same

Comment (AntonReact) [reply packages/auth/src/Login/ForceEnrollMfa.tsx:113]: @davidantoon 
There are several screens in the jira task. You can check out if the design is fine.
https://frontegg.atlassian.net/browse/FR-1170

**Pull request merged**